libbeat/outputs/elasticsearch/output.go

```diff
@@ -117,6 +118,10 @@ func (out *elasticsearchOutput) init(
 		return err
 	}
 
+	if len(clients) > 0 {
+		clients[0].Setup(config)
```

Comment:
you want to call ping with rather small timeout to check ES instance being available?